README.md

```diff
@@ -1,27 +1,168 @@
-# Role Name
+# Gitlab
```

Comment:
The role's name is `gitlab`:

![image](https://user-images.githubusercontent.com/4971975/35486106-591ceff2-0469-11e8-8221-7278af2f928f.png)

Reply:
fixed